### PR TITLE
Mobile styles

### PR DESCRIPTION
**Pull request by hys25**

### CODE REVIEW

**Comment by hys25** [comment]
Hi @ArthurGorbenko , here is mobile for whole website. Probably it still needs some improvements, I'll fix them when texts will be ready and contributed. Please review when you have time , thanks in advance

File: config/navRoutes.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  },
+  {
+    name: "projects",
+    path: "/",
+  },
+  {
+    name: "More about this website",
+    path: "/more-about-this-website",
+  },
+]
```
Comment (ArturHorbenko) [inline config/navRoutes.js:21]: you can spread `routes` in `routesMobile` to prevent repetition

**Review by ArturHorbenko — COMMENTED** [review]
Hi, great progress on mobile. I left one comment for simplification.

**Comment by ArturHorbenko** [comment]
I checked pages in the browsers. Overall looks great and modern! 
I'd recommend check mobile on safari ( there is redundant scroll and white border ) there. 
Once it'll be fixed - we can merge this branch

**Pull request merged**